code/modules/research/techweb/all_nodes.dm

```diff
@@ -786,11 +786,108 @@
+	description = "Equipment produced by Cybersun ostensibly for people other than just the Gorlex Hardliners."
+	boost_item_paths = list(/obj/item/faction_tech/hardliners)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/gorlex_republic
+	id = "ngr_tech"
+	display_name = "New Gorlex Republic Equipment"
+	description = "Equipment produced by the New Gorlex Republic."
+	boost_item_paths = list(/obj/item/faction_tech/gorlex_republic)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/cybersun
+	id = "cybersun_tech"
+	display_name = "Cybersun Equipment"
+	description = "Equipment produced by Cybersun."
+	boost_item_paths = list(/obj/item/faction_tech/cybersun)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/inteq
+	id = "inteq_tech"
+	display_name = "Inteq Risk Management Group Equipment"
+	description = "Equipment produced by the Inteq Risk Management Group."
+	boost_item_paths = list(/obj/item/faction_tech/inteq)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/clip
+	id = "clip_tech"
+	display_name = "Confederated League of Independent Planets Equipment"
+	description = "Equipment produced by the Confederated League of Independent Planets."
+	boost_item_paths = list(/obj/item/faction_tech/clip)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/pirate
+	id = "pirate_tech"
+	display_name = "Unlicensed Firearms"
+	description = "Unlicensed and possibly illegal firearm designs."
+	boost_item_paths = list(/obj/item/faction_tech/pirate)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/scarborough
+	id = "scarborough_tech"
+	display_name = "Scarborough Armaments Armaments"
+	description = "Armaments produced by Scarborough Armaments."
+	boost_item_paths = list(/obj/item/faction_tech/scarborough)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/srm
+	id = "srm_tech"
+	display_name = "Hunter's Pride Equipment"
+	description = "Equipment produced by Hunter's Pride."
+	boost_item_paths = list(/obj/item/faction_tech/srm)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/serene
+	id = "serene_tech"
+	display_name = "Serene Outdoors Equipment"
+	description = "Equipment produced by Serene Outdoors."
+	boost_item_paths = list(/obj/item/faction_tech/serene)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/solgov
+	id = "solgov_tech"
+	display_name = "The Most Serene Solar and Intersolar Confederation Equipment"
+	description = "Equipment produced by The Most Serene Solar and Intersolar Confederation."
+	boost_item_paths = list(/obj/item/faction_tech/solgov)
+	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 0)
+	hidden = TRUE
+
+/datum/techweb_node/SUNS
```

Comment:
```suggestion
/datum/techweb_node/suns
```
this should be lowercase it makes it easier to find (most lookups are case-sensitive and most [except for like two thinhs which drives me insane] are in all lowercase)